tools/deploy-static.sh

```diff
@@ -1,23 +1,37 @@
 #!/usr/bin/env bash
-# Script to prepare Docs for Cloudflare Pages deployment 
+
+# Script to prepare Docs for Cloudflare Pages deployment
 
 set -euo pipefail
 
-dirname=$(dirname $0)
+# resolve absolute path and account for spaces, etc
+dirname="$(cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P)"
```

Comment:
We can actually change this command in all the scripts, since many of them use this.

Reply:
True, but not something I wanted to pull into this PR. If they aren't broke...